lib/TWCManager/EMS/TeslaPowerwall2.py

```diff
@@ -219,10 +219,15 @@ def getPWJson(self, path):
                 )
                 r.raise_for_status()
             except Exception as e:
-                self.debugLog(
-                    4, "Error connecting to Tesla Powerwall 2 to fetch " + path
-                )
-                self.debugLog(10, str(e))
+                if hasattr(e,"response") and e.response.status_code == 403:
```

Comment:
I wanted to do `if e.response ...` but apparently error responses are falsy.  Simultaneously pythonic and confusing.